percona-xtradb-cluster/5.6/source/release-notes/Percona-XtraDB-Cluster-5.6.25-25.12.rst

```diff
@@ -0,0 +1,69 @@
+
+ Variable :variable:`wsrep_dirty_reads` now has Global scope as well. (PXC-373).
+ 
+ MTR coverage has been added to all tests in galera suite. Many bugs associated with mtr tests have been fixed.
+
+ A new variable :variable:`gcache.keep_pages_count`, analogous to :variable:`gcache.keep_pages_size`, has been added. The variable limits the number of overflow pages rather than the total memory occupied by all overflow pages. Whenever either of the variables are updated at runtime to a non-zero value, cleanup is called on excess overflow pages to delete them. This feature also fixes the bugs with integer overflow in the ``gcache`` module.
+
+ Updates have been made to wsrep code to ensure greater concordance with binary log and GTID so that failover of async slaves, among nodes of the cluster is seamless and consistent. To ensure this in :bug:`1421360`, all ``FLUSH`` commands (except ``FLUSH BINARY LOG`` and ``FLUSH LOGS``, and read lock-based flush such as ``FLUSH TABLES WITH READ LOCK`` and ``FLUSH TABLES FOR EXPORT``), ``ANALYZE TABLE``, Percona Server-specific flush statements for `user statistics <https://www.percona.com/doc/percona-server/5.6/diagnostics/user_stats.html>`_ and `page tracking bitmaps <https://www.percona.com/doc/percona-server/5.6/management/changed_page_tracking.html>`_ are executed under Total Order Isolation (TOI) so that they are replicated to other nodes in the cluster when they are written to binary log. 
+
+ |Percona XtraDB Cluster| has temporarily disabled savepoints in triggers and stored functions. The reason is that even having fixed bug :bug:`1438990` and bug :bug:`1464468` we have found more cases where savepoints in triggers break binary logging and replication, resulting in server crashes and broken slaves. This feature will be disabled until the above issues are properly resolved.
+
+Bugs fixed 
+==========
+
+ ``SHOW STATUS LIKE ...`` and ``SHOW STATUS`` were taking time proportional to size of :variable:`gcache.size`. Bug fixed :bug:`1462674`.
+
+ When disk space would get filled with :variable:`gcache.page` files, Galera would crash when the next page file was created. Bug fixed :bug:`1488535`.
+
+ XtraBackup SST didn't clobber :file:`backup-my.cnf` which caused SST to fail. Bug fixed :bug:`1431101`.
+
+ Error from ``process::wait`` was not checked in joiner thread leading to joiner starting erroneously even when SST had failed. Bug fixed :bug:`1402166`.
+
+ Fixes for bug :bug:`1407770` and bug :bug:`1426378` implemented in |Percona XtraDB Cluster| :rn:`5.6.24-25.11` introduced an regression. Update of the :variable:`wsrep_cluster_address` variable, following the update of :variable:`wsrep_provider_options` variable would cause the server deadlocked. Bug fixed PXC-421.
```

Comment:
Skip the "Fixes for bug :bug:`1407770` and bug :bug:`1426378` implemented in |Percona XtraDB Cluster| :rn:`5.6.24-25.11` introduced an regression.",  just  "Due to a regression introduced in in 5.6.24-25.11, update of the..." should be fine. (and 'to deadlock' is correct in the end)